libmscore/accidental.h

```diff
@@ -26,114 +26,111 @@
-      USER                // user created accidental
-      };
+    AUTO,                 // layout created accidental
+    USER                  // user created accidental
+};
 
-//---------------------------------------------------------
-//   AccidentalBracket
-//---------------------------------------------------------
+// ---------------------------------------------------------
+// AccidentalBracket
+// ---------------------------------------------------------
 
 enum class AccidentalBracket : char {
-      NONE,
-      PARENTHESIS,
-      BRACKET
-      };
+    NONE,
+    PARENTHESIS,
+    BRACKET
+};
 
-//---------------------------------------------------------
-//   SymElement
-//---------------------------------------------------------
+// ---------------------------------------------------------
+// SymElement
+// ---------------------------------------------------------
 
 struct SymElement {
-      SymId sym;
-      qreal x;
-      SymElement(SymId _sym, qreal _x) : sym(_sym), x(_x) {}
-      };
+    SymId sym;
+    qreal x;
+    SymElement(SymId _sym, qreal _x) : sym(_sym), x(_x) {}
+};
 
-//---------------------------------------------------------
-//   @@ Accidental
-//   @P role        enum  (Accidental.AUTO, .USER) (read only)
-//   @P small       bool
-//---------------------------------------------------------
+// ---------------------------------------------------------
+// @@ Accidental
+// @P role        enum  (Accidental.AUTO, .USER) (read only)
+// @P small       bool
+// ---------------------------------------------------------
 
-class Accidental final : public Element {
-      QList<SymElement> el;
-      AccidentalType _accidentalType { AccidentalType::NONE };
-      bool _small                    { false                   };
-      AccidentalBracket _bracket     { AccidentalBracket::NONE };
-      AccidentalRole _role           { AccidentalRole::AUTO    };
+class Accidental final : public Element
+{
+    QList<SymElement> el;
+    AccidentalType _accidentalType { AccidentalType::NONE };
+    bool _small                    { false                   };
+    AccidentalBracket _bracket     { AccidentalBracket::NONE };
+    AccidentalRole _role           { AccidentalRole::AUTO    };
 
-   public:
-      Accidental(Score* s = 0);
+public:
+    Accidental(Score* s = 0);
 
-      Accidental* clone() const override  { return new Accidental(*this); }
-      ElementType type() const override   { return ElementType::ACCIDENTAL; }
+    Accidental* clone() const override { return new Accidental(*this); }
+    ElementType type() const override { return ElementType::ACCIDENTAL; }
 
-      QString subtypeUserName() const;
-      void setSubtype(const QString& s);
-      void setAccidentalType(AccidentalType t)     { _accidentalType = t;    }
+    QString subtypeUserName() const;
+    void setSubtype(const QString& s);
+    void setAccidentalType(AccidentalType t)     { _accidentalType = t;    }
```

Comment:
The script didn't remove excessive whitespaces here, and left them after `;`. But it did for the next two lines which is odd.